### ISSUE TITLE
Add Saint Vincent and the Grenadines holidays

### ISSUE THREAD

**Issue by arkid15r** [dr-prodigy/python-holidays#1263]
Links:
  - https://en.wikipedia.org/wiki/Public_holidays_in_Saint_Vincent_and_the_Grenadines
  - https://www.timeanddate.com/holidays/saint-vincent-and-the-grenadines/
  - http://pmoffice.gov.vc/pmoffice/index.php/public-holidays

**Comment by waqar2403** [comment]
@arkid15r Hi I'm working on this issue and it's my first time working on holidays.
for now
- Went through the Contributing guide
- Forking and Local setup
- Checking some PRs and existing countries for reference
Is there anything I'm missing or any suggestions?

**Comment by arkid15r** [comment]
Hi @waqar2403
 
I suggest also checking [SPONSORSHIP.md](https://github.com/vacanza/holidays/blob/dev/SPONSORSHIP.md) closely (eligibility section). 

Good luck w/ your first contribution 👍

**Comment by waqar2403** [comment]
@arkid15r  I just went through the eligibility criteria can I work on two PRs in parallel one for sponsorship and another to fulfill the eligibility criteria?

**Comment by arkid15r** [comment]
As per the document `You must have at least one merged pull request that adds support for a new entity’s holidays.`
Please consider applying for July batch.

**Comment by waqar2403** [comment]
@arkid15r thanks for your clarification & Can I still work on this issue?

**Comment by arkid15r** [comment]
Yes, absolutely. Let me know if you're interested and I'll assign it to you.

**Comment by waqar2403** [comment]
@arkid15r  yeah I'm happy to work on it assign it to me

**Comment by waqar2403** [comment]
There is an issue in the Carnival Dates for Other Caribbean and SVG, SVG shifted their dates to the 2nd week of July which is different from all other countries Any suggestions how to handle it? (solved)
